networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -20,7 +20,7 @@ Before you can use NMState with {product-title}, you must install the Kubernetes
 ====
 The Kubernetes NMState Operator updates the network configuration of a secondary NIC. The Operator cannot update the network configuration of the primary NIC, or update the `br-ex` bridge on most on-premise networks.
 
-On a bare-metal platform, using the Kubernetes NMState Operator to update the `br-ex` bridge network configuration is only supported if you set the `br-ex` bridge as the interface in a machine config manifest file. To update the `br-ex` bridge as a postinstallation task, you must set the `br-ex` bridge as the interface in the NMState configuration of the `NodeNetworkConfigurationPolicy` custom resource (CR) for your cluster.
+On a bare-metal platform, using the Kubernetes NMState Operator to update the `br-ex` bridge network configuration is only supported if you set the `br-ex` bridge as the interface in a machine config manifest file. To update the `br-ex` bridge as a postinstallation task, you must set the `br-ex` bridge as the interface in the NMState configuration of the `NodeNetworkConfigurationPolicy` custom resource (CR) for your cluster. For more information, see xref:../../installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc#creating-manifest-file-customized-br-ex-bridge_ipi-install-post-installation-configuration[Creating a manifest object that includes a customized br-ex bridge] in _Setting up the environment for an OpenShift installation_.
```

Comment:
```suggestion
On a bare-metal platform, using the Kubernetes NMState Operator to update the `br-ex` bridge network configuration is only supported if you set the `br-ex` bridge as the interface in a machine config manifest file. To update the `br-ex` bridge as a postinstallation task, you must set the `br-ex` bridge as the interface in the NMState configuration of the `NodeNetworkConfigurationPolicy` custom resource (CR) for your cluster. For more information, see xref:../../installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration.adoc#creating-manifest-file-customized-br-ex-bridge_ipi-install-post-installation-configuration[Creating a manifest object that includes a customized br-ex bridge] in "Setting up the environment for an OpenShift installation".
```

Reply:
I _think_ in ISG we are suppose to use " " for sentence style capitalization when citing internal sources/titles. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks#double-quotation-marks:~:text=Citations%20that%20have%20sentence%2Dstyle%20capitalization%20and%20do%20not%20have%20any%20other%20highlighting

Reply:
Thanks, Joe. I followed our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-xref-attribute). 

![Screenshot from 2024-08-22 11-19-07](https://github.com/user-attachments/assets/85150057-eef4-4f3d-80f8-122920a33a08)

Reply:
I think the RH SSG trumps the IBM according to this [excellent pyramid graphic](https://redhat-documentation.github.io/supplementary-style-guide/#_using_style_guides_for_redhat_product_documentation) on the RH SSG.